hack/install.sh

```diff
@@ -34,6 +34,22 @@ pgp.mit.edu
 keyserver.ubuntu.com
 "
 
+environment=''
+while getopts "e:" opt; do
+        case $opt in
+                e)
```

Comment:
I wonder if the option should have a more descriptive name (for example, `-mirror`?) @tianon wdyt?